build.gradle.kts

```diff
@@ -87,3 +88,18 @@ jsonSchema2Pojo {
     generateBuilders.set(true)
     annotationStyle.set("gson")
 }
+
+
+
+publishing {
+    publications {
+        create<MavenPublication>("maven") {
+            groupId = "dev.sigstore"
+            artifactId = "sigstore-clients"
```

Comment:
`sigstore-clients` -> `sigstore-java` for consistency with repo name?
You can also use: `artifactId = project.name`